compositor_render/src/transformations/shader_header.wgsl

```diff
@@ -1,14 +1,16 @@
 struct VertexInput {
     @location(0) position: vec3<f32>,
     @location(1) tex_coords: vec2<f32>,
+    @location(2) video_id: i32,
```

Comment:
I know that in the previous PR I used multiple, inconsistent names for this field (my bad. Now when we validate it, we should agree on a single, suitable name. 
I would prefer to use the `texture_id` name, since this id "tells" to which "texture plane" vertex belongs. This texture doesn't have to be video (it can be an image, text, basically anything node can produce). `input_id` on the other hand can be confused with compositor input streams.

Reply:
Remember to rename it in other shaders as well

Reply:
sure I wasn't sure which was the prefered way, video_id was in more places so I assumed that was the intention